files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```diff
@@ -3696,7 +3697,28 @@ tools:
       - interactive_pulsar
       accept:
       - training-exempt
+  restricted_interactive_tool:
+  inherits: interactive_tool_.*
+  rules:
+    - id: restricted_interactive_tool_restriction_rule
+      if: |
+        from datetime import datetime, timedelta
+        interactive_tool_embargo_days = 28
+        may_use_ITs = False
+        if 'Trusted_for_ITs' in [r.name for r in user.all_roles() if not r.deleted]:
+          may_use_ITs = True
+        if any([r.name.startswith('training') for r in user.all_roles()]):
+          may_use_ITs = True
+        threshold = datetime.now() - timedelta(days=interactive_tool_embargo_days)
+        if user.create_time < threshold:
+          may_use_ITs = True
```

Comment:
```suggestion
        may_use_ITs = False
        threshold = datetime.now() - timedelta(days=interactive_tool_embargo_days)
        if 'Trusted_for_ITs' in [r.name for r in user.all_roles() if not r.deleted]:
          may_use_ITs = True
        elif any([r.name.startswith('training') for r in user.all_roles()]):
          may_use_ITs = True
        elif user.create_time < threshold:
          may_use_ITs = True
```

Reply:
I agree. I just fixed the indentation so I'll apply this